cloudinit/net/openbsd.py

```diff
@@ -54,6 +54,12 @@ def set_route(self, network, netmask, gateway):
             fn = subp.target_path(self.target, if_file)
             content = gateway + "\n"
             util.write_file(fn, content)
+        else:
+            try:
```

Comment:
It would be more elegant to define the attribute in the constructor.

Reply:
Sure, I'm aware of this being hacky. But tbh, I don't know how to do this right -- I hardly ever had to do anything in python yet.